### PR TITLE
Pull request for WAZO-2124-restrict-master-tenant-resources

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+  * GET `/1.1//provisioning/networking`
+  * PUT `/1.1//provisioning/networking`
+  * POST `/1.1//sip/transports`
+  * PUT `/1.1//sip/transports/<transport_uuid>`
+  * DELETE `/1.1//sip/transports/<transport_uuid>`
+  * GET `/1.1//registrars`
+  * POST `/1.1//registrars`
+  * GET `/1.1//registrars/<registrar_uuid>`
+  * PUT `/1.1//registrars/<registrar_uuid>`
+  * DELETE `/1.1//registrars/<registrar_uuid>`
```
Comment (bloom1) [inline CHANGELOG.md:63]: You've got double slashes in all of these routes

Comment (fblackburn1) [reply CHANGELOG.md]: oups, fixed

**Review by bloom1 — APPROVED** [review]

**Review by bloom1 — CHANGES_REQUESTED** [review]
I cancelled the Zuul job.
I saw something last minute:
In the confd logs, I have
```
(DEBUG) (wazo_lib_rest_client.client): AuthClient received unexpected arguments: ['master_tenant_uuid'] 
```

**Comment by fblackburn1** [comment]
> I cancelled the Zuul job.
> I saw something last minute:
> In the confd logs, I have
> 
> ```
> (DEBUG) (wazo_lib_rest_client.client): AuthClient received unexpected arguments: ['master_tenant_uuid'] 
> ```

Well I suggest to open another ticket, because this behavior is in amid/plugind too. Since it's only a debug message, there is no real issue

**Comment by bloom1** [comment]
> > I cancelled the Zuul job.
> > I saw something last minute:
> > In the confd logs, I have
> > ```
> > (DEBUG) (wazo_lib_rest_client.client): AuthClient received unexpected arguments: ['master_tenant_uuid'] 
> > ```
> 
> Well I suggest to open another ticket, because this is the same behavior than amid/plugind (and other non-public repos). Since it's only a debug message, there is no real issue

Alright, I just wanted to make sure :+1:

**Review by bloom1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Comment by bloom1** [comment]
recheck

**Pull request merged**